blkn/src/Button/__tests__/index.test.js

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import { shallow } from "enzyme";
+import Button from "../";
+
+describe("Button", () => {
```

Comment:
this whole thing is unnecessarily complicated.. use local variables instead of reassigning, remove the useless before each, don't nest describes so much

Reply:
I simplified the tests.